deployment/sw360baseimage/setupProxy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+# Copyright Bosch Software Innovations GmbH, 2016.
+# Part of the SW360 Portal Project.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+# this script does
+#   - unset the proxies, if they were set to "None"
+#   - set the proxies to "${gateway}:3128", if they were set to CNTLM_ON_GATEWAY
+
+if [ "$http_proxy" ]; then
+    if [ "$http_proxy" = "None" ]; then
+        unset http_proxy
+    elif [ "$http_proxy" = "CNTLM_ON_GATEWAY" ]; then
+        export "http_proxy"="http://$(ip route get 8.8.8.8 | awk '{ print $3 }'):3128"
+    fi
+    echo "http_proxy is set to $http_proxy"
+fi
+
+if [ "$https_proxy" ]; then
```

Comment:
This is just code duplication and sooner or later someone will make adaptations and forget to adapt the duplicated part.
Why not do something like:
```
for proto in http https                                                     
 do
 varname=${proto}_proxy
 if [ ${!varname} ]; then
 echo foo
 else
 echo bar
 export ${varname}="${proto}://${default_gw}:${port}"
 fi
 done
```